packages/@storefront/structure/src/carousel/index.ts

```diff
@@ -28,6 +28,9 @@ class Carousel {
 
   onMount() {
     utils.WINDOW().addEventListener('resize', this.forceUpdate);
+
+    // the items have a width of 0 if items are simply set on init, so do an update to recalculate styles
+    this.set(true);
```

Comment:
@GregYaniw I would swap the `set()` call out for `this.forceUpdate()`, which more clearly expresses your intention. That said, I would still leave the comment in place.